### ISSUE TITLE
Slider element default styles broken on android

### ISSUE THREAD

**Issue by n1ru4l** [react-native-community/react-native-elements#248]
```jsx
import React from 'react'
import { View, StyleSheet } from 'react-native'

import { Slider } from 'react-native-elements'

function PlaybackControls() {
  return (
    <View style={Style.PlaybackControls}>
      <Slider
        style={Style.PlayBackControls__Slider}
      />
    </View>
  )
}

const Style = StyleSheet.create({
  PlaybackControls: {},
  PlayBackControls__Slider: {
    width: `90%`,
  },
})
```

![debug](https://cloud.githubusercontent.com/assets/14338007/24321033/cac7626a-1142-11e7-9ec5-1b8391379427.png)

I also tried to adjust the styles myself by setting the `trackStyle` and `thumbStyle` property:

```jsx 
import React from 'react'
import { View, StyleSheet } from 'react-native'

import { Slider } from 'react-native-elements'

function PlaybackControls() {
  return (
    <View style={Style.PlaybackControls}>
      <Slider
        style={Style.PlayBackControls__Slider}
        trackStyle={Style.PlayBackControls__SliderTrack}
        thumbStyle={Style.PlayBackControls__SliderThumb}
      />
    </View>
  )
}

const Style = StyleSheet.create({
  PlaybackControls: {},
  PlayBackControls__Slider: {
    width: `90%`,
    height: 20,
    overflow: `hidden`,
  },
  PlayBackControls__SliderTrack: {
    borderRadius: 0,
    height: `100%`,
  },
  PlayBackControls__SliderThumb: {
    top: `100%`,
    height: 20,
    borderRadius: 0,
    width: 10,
  },
})
```
There is always some gap between the thumb and the track I can't get rid of.
![debug2png](https://cloud.githubusercontent.com/assets/14338007/24321094/1a3282b6-1144-11e7-81d2-974dc91b99aa.png)

**Edit:**

I fixed it with the following styles ( feels a bit hacky :/ ):
```jsx
const Style = StyleSheet.create({
  PlaybackControls: {},
  PlayBackControls__Slider: {
    width: `100%`,
    height: 20,
    marginBottom: -1,
  },
  PlayBackControls__SliderTrack: {
    borderRadius: 0,
    height: 19,
    marginTop: 0,
  },
  PlayBackControls__SliderThumb: {
    top: `100%`,
    height: 20,
    borderRadius: 0,
    width: 10,
    marginTop: -1,
  },
})
```

**Comment by LoriKarikari** [comment]
I was having the same issue on iOS and Android.
![selection_010](https://cloud.githubusercontent.com/assets/7902980/24454151/15aa7722-148b-11e7-9a8c-15c3c6ccb7cb.png)

I fixed it with the following styles: 
```
<Slider
  thumbStyle={styles.thumb}
  trackStyle={styles.track}
  ...
 />

const styles = StyleSheet.create({
  track: {
      marginTop: 0,
    },
  thumb: {
      top: 20
    }
})
```
![selection_009](https://cloud.githubusercontent.com/assets/7902980/24454155/1a166d5c-148b-11e7-8476-ab86a6a7dc7c.png)

**Comment by Monte9** [comment]
@LoriKarikari That's great! Would be great if you could submit a PR for it. I can get it merged in ASAP. Thanks.

**Comment by LoriKarikari** [comment]
Okay will do.

**Comment by Monte9** [comment]
This issue has been fixed on PR #252. Thanks @LoriKarikari 

I just published v0.10.3 which should have this fix.
